Sources/Commands/UserToolchain.swift

```diff
@@ -122,33 +122,18 @@ public struct UserToolchain: Toolchain {
         return (SWIFT_EXEC ?? resolvedBinDirCompiler, SWIFT_EXEC_MANIFEST ?? resolvedBinDirCompiler)
     }
 
-    public init(destination: Destination) throws {
-        self.destination = destination
-
-        // Get the search paths from PATH.
-        let envSearchPaths = getEnvSearchPaths(
-            pathString: getenv("PATH"), currentWorkingDirectory: localFileSystem.currentWorkingDirectory)
-
-        func lookup(fromEnv: String) -> AbsolutePath? {
-            return lookupExecutablePath(
-                filename: getenv(fromEnv),
-                searchPaths: envSearchPaths)
-        }
-
-        // Get the binDir from destination.
-        let binDir = destination.binDir
+    private static func lookup(fromEnv: String, searchPaths: [AbsolutePath]) -> AbsolutePath? {
```

Comment:
I think we should rename "fromEnv" to "executable", so the signature becomes:

```swift
private static func lookup(executable: String, searchPaths: [AbsolutePath]) -> AbsolutePath?
````

Reply:
actually, it should be "variable" not "executable".